login.php

```diff
@@ -0,0 +1,20 @@
+<?php
+session_start ();
+$session = $_SESSION ['logged-in'];
+
+if ($session === 1) {
```

Comment:
`if ( $_SESSION['logged-in'] === 1)`

Reply:
Yes that would make more sense but for some reason it breaks it if i change it. I'll leave it for now but investigate further.